### PR TITLE
Flashable switch added

### PR DESCRIPTION
**Pull request by chanharryhk**
# Description
To enable or disable flashloans
# Integrations Checklist

- [ ] Have any function signatures changed? If yes, outline below.
- [ ] Have any features changed or been added? If yes, outline below.
- [ ] Have any events changed or been added? If yes, outline below.
- [ ] Has all documentation been updated?

# Changelog

## Events

### CODE REVIEW

**Comment by chanharryhk** [comment]
@tommy4241 requested cross platform flashloan pause will make the changes shortly

**Review by kafitz — APPROVED** [review]

File: src/Curve.sol [hunk 1]
```diff
@@ -320,6 +320,11 @@ contract Curve is Storage, MerkleProver, NoDelegateCall {
         _;
     }
 
+    modifier isFlashable() {
+        require(ICurveFactory(address(curveFactory)).getFlashableState(), "Curve/flashloans-paused");
```
Comment (chanharryhk) [inline src/Curve.sol:324]: @tommy4241 is this bad practice?

Comment (tommy4241) [reply src/Curve.sol:324]: this is fine @chanharryhk

**Review by tommy4241 — APPROVED** [review]
looks great

**Pull request merged**